content/about/feature-stages/index.md

```diff
@@ -57,11 +60,12 @@ Below is our list of existing features and their current phases. This informatio
 | [Client and Server Telemetry Reporting](/docs/concepts/policies-and-telemetry/) | Stable
 | [Service Dashboard in Grafana](/docs/tasks/telemetry/metrics/using-istio-dashboard/) | Stable
 | [Istio Component Dashboard in Grafana](/docs/tasks/telemetry/metrics/using-istio-dashboard/) | Stable
+| [Distributed Tracing to Zipkin / Jaeger](/docs/tasks/telemetry/distributed-tracing/) | Stable
+| [Service Tracing](/docs/tasks/telemetry/distributed-tracing/) | Stable
 | [Stackdriver Integration](/docs/reference/config/policy-and-telemetry/adapters/stackdriver/) | Beta
-| [Distributed Tracing to Zipkin / Jaeger](/docs/tasks/telemetry/distributed-tracing/) | Beta
-| [Service Tracing](/docs/tasks/telemetry/distributed-tracing/) | Beta
 | [Logging with Fluentd](/docs/tasks/telemetry/logs/fluentd/) | Beta
 | [Trace Sampling](/docs/tasks/telemetry/distributed-tracing/overview/#trace-sampling) | Beta
+| Prometheus Operator | Alpha
```

Comment:
I'd maybe call this `Prometheus Operator Support`.

Reply:
is this new?  I don't recall a doc page for it, neither in my k8s cluster when istio default profile is installed @douglas-reid

Reply:
@howardjohn reminded me this is for new installer, so i think we should not mention this for 1.2

Reply:
Removed mention of the Prometheus Operator